.env

```diff
@@ -18,6 +18,7 @@ TRUSTED_PROXIES=
 
 ###> symfony/framework-bundle ###
 APP_ENV=dev
+APP_DEBUG=true
```

Comment:
Dat is niet verstandig definitief door te voeren, dit wordt dan ook actief op prod

Reply:
Is verwijderd.